runtime/parachains/src/runtime_api_impl/v1.rs

```diff
@@ -232,18 +232,26 @@ pub fn session_index_for_child<T: initializer::Config>() -> SessionIndex {
 	<shared::Module<T>>::session_index()
 }
 
-pub fn get_historical_authority_ids<T: initializer::Config>() -> Vec<AuthorityDiscoveryId> {
+/// Implementation for the `AuthorityDiscoveryApi::authorities()` function of the runtime API.
+/// Gets current and historical authority ids using session_info module.
```

Comment:
we should also mention that this call can be heavy, however for the purposes of the authority-discovery it's not critical